### ISSUE TITLE
[UWP] Stringify JSON of card isn't correct

### ISSUE THREAD

**Issue by michael-hawker** [Microsoft/AdaptiveCards#1808]
# Platform

What platform is your issue or question related to? (Delete other platforms).
* UWP

# Author or host

Author, rendering with UWP lib.

# Version of SDK

NuGet 1.0.1

# Issue
If I use a ShowCard example, I can use 'popup' mode to get the card object of the inner show command.  I can render this separately if I want passing the card object to RenderAdaptiveCard.  However, if I instead use .ToJson().Stringify() on the card instead, it doesn't give me something that I can parse back into a card (and it is missing info from the card like the body).

```
{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "type": "AdaptiveCard",
  "version": "1.0",
  "body": [
    {
      "type": "TextBlock",
      "text": "This card's action will show another card"
    }
  ],
  "actions": [
    {
      "type": "Action.ShowCard",
      "title": "Action.ShowCard",
      "card": {
        "type": "AdaptiveCard",
        "body": [
          {
            "type": "TextBlock",
            "text": "What do you think?"
          }
        ],
        "actions": [
          {
            "type": "Action.Submit",
            "title": "Neat!"
          }
        ]
      }
    }
  ]
}
```

```
{"actions":[{"data":"","id":"","title":"Neat!","type":"Action.Submit"}],"backgroundImage":"","body":[],"fallbackText":"","lang":"","speak":"","type":"AdaptiveCard","version":""}
```

**Comment by RebeccaAnne** [comment]
Two issues here. 

- The 1.0 serialization code had some bugs in it which resulted in cards not serializing properly. These were fixed by #1434 which isn't in 1.0.1. 

- Top level card requires the version property, whereas a show card does not. As a result, when we roundtrip the show card as a stand-alone card through the ToJson/FromJson code we end up with an error due to the card not having version set. We should probably set a valid version on the show card when we serialize it to address this issue.

**Comment by RebeccaAnne** [comment]
Fixed by #1875
